### ISSUE TITLE
Pagination: Number of users per page dropdown label is not announced with tab key navigation

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4668]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 4.1.2 Name, Role, Value

**Test Environment:**
- Test URL: https://clarity.design/documentation/datagrid/pagination
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open the test URL.
2. Select some users or press select all checkbox.
3. Navigate to &quot;Users per page&quot; dropdown and observe screen reader&#039;s spoken feedback.

**Expected result:**
 Appropriate Name, Role, Value should be announced.

**Actual result:**
Only Role and value is announced. i.e. label is not announced.	

Reported by @gkhadK

**Comment by balajun01** [comment]
is there any way to implement the only pagination footer with my table content
